lib/msal-browser/src/controllers/UnknownOperatingContextController.ts

```diff
@@ -0,0 +1,530 @@
+    }
+
+    getAccountByHomeId(homeAccountId: string): AccountInfo | null {
+        this.logger.trace("getAccountByHomeId called");
+        if (!homeAccountId) {
+            this.logger.warning(
+                "getAccountByHomeId: No homeAccountId provided"
+            );
+            return null;
+        }
+
+        const account = this.browserStorage.getAccountInfoFilteredBy({
+            homeAccountId,
+        });
+        if (account) {
+            this.logger.verbose(
+                "getAccountByHomeId: Account matching homeAccountId found, returning"
+            );
+            this.logger.verbosePii(
+                `getAccountByHomeId: Returning signed-in accounts matching homeAccountId: ${homeAccountId}`
+            );
+            return account;
+        } else {
+            this.logger.verbose(
+                "getAccountByHomeId: No matching account found, returning null"
+            );
+            return null;
+        }
+    }
+    getAccountByLocalId(localAccountId: string): AccountInfo | null {
+        this.logger.trace("getAccountByLocalId called");
+        if (!localAccountId) {
+            this.logger.warning(
+                "getAccountByLocalId: No localAccountId provided"
+            );
+            return null;
+        }
+
+        const account = this.browserStorage.getAccountInfoFilteredBy({
+            localAccountId,
+        });
+        if (account) {
+            this.logger.verbose(
+                "getAccountByLocalId: Account matching localAccountId found, returning"
+            );
+            this.logger.verbosePii(
+                `getAccountByLocalId: Returning signed-in accounts matching localAccountId: ${localAccountId}`
+            );
+            return account;
+        } else {
+            this.logger.verbose(
+                "getAccountByLocalId: No matching account found, returning null"
+            );
+            return null;
+        }
+    }
+    getAccountByUsername(username: string): AccountInfo | null {
+        this.logger.trace("getAccountByUsername called");
+        if (!username) {
+            this.logger.warning("getAccountByUsername: No username provided");
+            return null;
+        }
+
+        const account = this.browserStorage.getAccountInfoFilteredBy({
+            username,
+        });
+        if (account) {
+            this.logger.verbose(
+                "getAccountByUsername: Account matching username found, returning"
+            );
+            this.logger.verbosePii(
+                `getAccountByUsername: Returning signed-in accounts matching username: ${username}`
+            );
+            return account;
+        } else {
+            this.logger.verbose(
+                "getAccountByUsername: No matching account found, returning null"
+            );
+            return null;
+        }
+    }
+    getAllAccounts(): AccountInfo[] {
+        BrowserUtils.blockAPICallsBeforeInitialize(this.initialized);
```

Comment:
Why is this blocked?

Reply:
I think this was added recently.  And I wasn't familiar with how to implement it.

Reply:
Fixed.